CONTRIBUTING.md

```diff
@@ -100,12 +100,12 @@ Please follow the following steps for every pull request to ensure every contrib
 
 1. Create a [Draft Pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/) including a description on what will be changed and what impact you expect. The title can start with "[Draft] " during development.
 2. Work on the Pull request
-3. Before a Pull request can be considered ready all upstream changes from the `develop` branch need to be merged into it. It is the duty of everyone to help make merging into `develop` as painless as possible so please try to align if you see that your Pull requests works in a similar area as another one.
-4. Once the Pull request is ready, fulfills all requirements outlined above and is up to date with the `develop` branch it can be converted from Draft and marked as ready for review.
-5. Having multiple reviews for every Pull request would be nice. Reviews from community members not mentioned in the [CODEOWNERS](CODEOWNERS) file should be taken serious and used as an important source of feedback but have no decisional power on what gets merged into the `develop` branch.
+3. Before a Pull request can be considered ready all upstream changes from the `community` branch need to be merged into it. It is the duty of everyone to help make merging into `community` as painless as possible so please try to align if you see that your Pull requests works in a similar area as another one.
+4. Once the Pull request is ready, fulfills all requirements outlined above and is up to date with the `community` branch it can be converted from Draft and marked as ready for review.
```

Comment:
Add comma here after the word above:

"fulfills all requirements outlined above,"

add comma after branch "branch, it can be converted ..."